arm/Microsoft.RecoveryServices/vaults/replicationFabrics/replicationProtectionContainers/deploy.bicep

```diff
@@ -13,7 +13,7 @@ param replicationContainerMappings array = []
 @description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
 param enableDefaultTelemetry bool = true
 
-var enableReferencedModulesTelemetry = false
+var enableChildTelemetry = false
```

Comment:
Please undo

Reply:
This change was suggested by eriqua in old PR

Reply:
It's actually the other way around. The `enableReferencedModulesTelemetry` is the latest version

Reply:
Hi @prasanjeets, I agree with @MrMCake, the correct variable name to use is `enableReferencedModulesTelemetry`.

Reply:
Done